configure

```diff
@@ -580,6 +580,7 @@ fi
 BOOL_OPTIONS=""
 VAL_OPTIONS=""
 
+opt option-checking 1 "complain about unrecognized options"
```

Comment:
Of course. Should I move it at the end of all options or just after most `--enable/--disable`-style options (before `--localstatedir`) ?